plugins/pixart-rf/fu-pxi-ble-device.c

```diff
@@ -803,6 +803,8 @@ fu_pxi_ble_device_write_firmware(FuDevice *device,
 	if (!fu_pxi_ble_device_reset(self, error))
 		return FALSE;
 	fu_progress_step_done(progress);
+	fu_device_add_flag(device, FWUPD_DEVICE_FLAG_WAIT_FOR_REPLUG);
+	fu_device_set_remove_delay(device, 10000);
```

Comment:
Can you set move just fu_device_set_remove_delay() into fu_pxi_ble_device_init() please -- I think the daemon will complain otherwise.

Reply:
> Can you set move just fu_device_set_remove_delay() into fu_pxi_ble_device_init() please -- I think the daemon will complain otherwise.

Fixed https://github.com/fwupd/fwupd/commit/4ecd3365c65835d99e878c3b69afe0235f394d36